epub34/a11y/index.html

```diff
@@ -1499,15 +1486,15 @@ <h4>Re-evaluating conformance</h4>
 			<section id="sec-feedback">
 				<h3>Feedback</h3>
 
-				<p>Although EPUB creators are expected to assess their publications prior to certifying them accessible,
+				<p>Although publishers are expected to assess their publications prior to certifying them accessible,
```

Comment:
Not again! And so soon. Maybe something like:

```suggestion
				<p>While it is expected that publications will be assessed prior to certifying them accessible, 
```